bin/kano-settings-onboot

```diff
@@ -217,6 +219,22 @@ def ensure_correct_volume_levels(dry_run=False):
     # return info
 
 
+def ensure_screen_kit_rotation_global():
+    """
+    Ensure the screen kit orientation setting is also set globally.
+
+    This sets a 'fallback flag' in the unlikely situation where a SK will
+    lose its EDID data. No reboot is required since the option is already set
+    in the config.txt by default - this just mirrors it.
+    """
+    if not get_config_value('display_rotate') and is_screen_kit():
```

Comment:
`is_screen_kit` uses EDID. What if that's already corrupted?

Reply:
If the EDID is not there even at first boot, I'm throwing in the towel. It shouldn't pass manufacturing QA!

Reply:
As per verbal discussion - there is - we could check the USB.